rpm/gratia-probe.spec

```diff
@@ -310,27 +312,32 @@ fi
 # %%{default_prefix}/gratia/common2/pginput.py
 # %%{default_prefix}/gratia/common2/probeinput.py
 
-%package condor
+%package condor-ap
 Summary: A Condor probe
 Group: Applications/System
 Requires: %{name}-common = %{version}-%{release}
 Requires: condor
 Requires: %{condor_python}
+Provides: %{name}-condor = %{version}-%{release}
+Provides: %{name}-glideinwms = %{version}-%{release}
+Obsoletes: %{name}-condor < 2.3.0
+Obsoletes: %{name}-glideinwms < 2.3.0
 
-%description condor
+%description condor-ap
 The Condor probe for the Gratia OSG accounting system.
 
-%files condor
+%files condor-ap
 %defattr(-,root,root,-)
-%doc %{default_prefix}/gratia/condor/README
-%dir %{default_prefix}/gratia/condor
-%{default_prefix}/gratia/condor/condor_meter
+%doc %{default_prefix}/gratia/condor-ap/README
+%dir %{default_prefix}/gratia/condor-ap
+%{default_prefix}/gratia/condor-ap/condor_meter
 %config(noreplace) %{_sysconfdir}/condor/config.d/50-gratia.conf
-%config(noreplace) %verify(not md5 size mtime) %{_sysconfdir}/gratia/condor/ProbeConfig
-%config(noreplace) %{_sysconfdir}/cron.d/gratia-probe-condor.cron
+%config(noreplace) %{_sysconfdir}/condor/config.d/50-gratia-gwms.conf
+%config(noreplace) %verify(not md5 size mtime) %{_sysconfdir}/gratia/condor-ap/ProbeConfig
```

Comment:
It'd be good to compare what we ship vs what @rynge actually wants https://github.com/opensciencegrid/osg-flock/blob/master/rpm/ProbeConfig-OSPool.

Though I imagine that we'll want to change a bunch of the dirs to accommodate a `SCHEDD_CRON` like what we have to do for the CE probe.